news/1879.bugfix

```diff
@@ -0,0 +1 @@
+Do not hardcode show_inactive in search; let CMFPlone handle that.
```

Comment:
Sprinkle some reStructuredText syntax for `CHANGES.rst`, and give yourself credit.

```suggestion
Do not hardcode ``show_inactive`` in search; let ``Products.CMFPlone`` handle that. @gyst
```